### DISCUSSION TITLE
Modal with Left Action - Component in Sketch vs on Paste site

### DISCUSSION THREAD

**Discussion by ang-twilio** [twilio-labs/paste#907] [Q&A] [answered]
Hi all,
I'm creating a modal that is using the left action for a back button. On the Paste site, I see the [left button as a secondary button](https://paste.twilio.design/components/modal/#footer-actions-directional-alignment), but when I am creating the modal in Sketch, it automatically uses a link-style button. When I try to change it to a secondary button, it is misaligned and I can't fix it unless I detach the symbol. Is the secondary button the recommended pattern or can we use either button style for a back button? Thanks!

![image](https://user-images.githubusercontent.com/71658977/98052800-03501780-1dec-11eb-8f8f-ed982804df43.png)

**Comment by serifluous** [comment, marked as answer]
Hey @ang-twilio ! We must've reverted some layer pinning on Buttons. Will merge in a fix soon.

**Reply by serifluous** [reply]
@ang-twilio Just merged in a fix for this!

**Reply by ang-twilio** [reply]
Working as expected, thanks Sarah!
